js/constants/constants.js

```diff
@@ -254,7 +257,8 @@ module.exports = {
     ENGLISH: "en",
     SPANISH: "es",
     CHINESE: "zh",
-    JAPANESE: "ja"
+    JAPANESE: "ja",
+    NOT_MATCHED: "und"
```

Comment:
@jpfranco @aeng7 Do we have a different value that we use when it's undefined usually?  I remember something about 'zz' but maybe that's not related to this.

Reply:
Hmm, not sure. I think we also need to add Korean though

Reply:
There's a section in [ISO-639-2 faq](https://www.loc.gov/standards/iso639-2/faq.html#25) and it actually mentions  `zxx (No linguistic content; Not applicable)`, but I think our case is more about
> If there is language content, but the specific language cannot be determined a special identifier is provided by ISO 639-2  **und (Undetermined)**

Although I think it could be better to change the constant to `UNDETERMINED`.